app/Http/Controllers/Web/ProfileSubscriptionsController.php

```diff
@@ -52,7 +52,13 @@ public function update(Request $request)
 
         $currentMobile = $user->mobile;
 
-        $this->registrar->register($request->all(), $user, function (
+        // Make sure that we wipe out existing email topics if the form is submitted with none checked
+        $request = $request->all();
+        if (!array_key_exists('email_subscription_topics', $request)) {
+            $request['email_subscription_topics'] = [];
+        }
+
+        $this->registrar->register($request, $user, function (
```

Comment:
Nit -- maybe worth naming this `$input` (like in the [Laravel docs](https://laravel.com/docs/7.x/requests#retrieving-input) example usage) to avoid overriding the `$request` itself?
```suggestion
        // Make sure that we wipe out existing email topics if the form is submitted with none checked
        $input = $request->all();
        if (!array_key_exists('email_subscription_topics', $input)) {
            $input['email_subscription_topics'] = [];
        }

        $this->registrar->register($input, $user, function (
```

Reply:
LOVE that idea! I was actually concerned about overriding the `$request` too (even though we're not using it again as of now) for future confusion but wasn't sure about naming and this solves it!

Reply:
Just pushed up this change! https://github.com/DoSomething/northstar/pull/1178/commits/cce3a3393c99d69bc46e8c95d2fa1a2d13d6ebde